groups/staffware-db/variables.tf

```diff
@@ -234,3 +234,12 @@ variable "netapp_ips" {
   type        = list(string)
   description = "List of Netapp IP addresses to use for iscsi discovery."
 }
+
+# ------------------------------------------------------------------------------
+# CHIPS DB SG variables
+# ------------------------------------------------------------------------------
+variable "chips_db_sg" {
```

Comment:
same as above for chips -> staffware naming

Reply:
changed to description = "List of Staffware DB Security Groups"